### PR TITLE
update tutorial lockfile

### PR DESCRIPTION
**Pull request by Manishearth**
Caught when running tutorial doctests

### CODE REVIEW

File: docs/tutorials/crates/Cargo.lock [hunk 1]
```diff
@@ -2796,6 +2695,22 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "time"
+version = "0.3.20"
```
Comment (sffc) [inline docs/tutorials/crates/Cargo.lock:2700]: Note that 0.3.20 violates our msrv

File: docs/tutorials/crates/Cargo.lock [hunk 2]
```diff
@@ -226,50 +268,68 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
...
- "ansi_term",
- "atty",
+ "clap_builder",
+ "clap_derive",
+ "once_cell",
+]
+
+[[package]]
+name = "clap_builder"
+version = "4.2.2"
```
Comment (sffc) [inline docs/tutorials/crates/Cargo.lock:292]: Note that 4.2.2 violates our msrv

**Review by sffc — COMMENTED** [review]
Hmm, I did update the tutorial lockfile but I think I may have done it in a different mode (note that we have both patch-local and cratesio mode). There are some deps in here that violate msrv, which I guess is okay since we don't run the msrv test on these crates?

**Comment by Manishearth** [comment]
Yeah I think that's okay. I don't know why I got something different, I'm happy to close this and let you investigate later

**Review by sffc — APPROVED** [review]
Hmm, actually it looks like it did get updated in https://github.com/unicode-org/icu4x/commit/14891059ee6c7e379b39124f773361487c8a0a8c#diff-ffffa7d1cc4954b13362bcfdc8b61152919b4766b601baf555d0acec2dd1f807, but then it was reverted in https://github.com/unicode-org/icu4x/commit/bf4d143e8e33a8fc81102a5580932cb630999924#diff-ffffa7d1cc4954b13362bcfdc8b61152919b4766b601baf555d0acec2dd1f807, so this puts it back to where it should be now.

**Pull request merged**